examples/data_types_and_io/data_types_and_io/structured_dataset.py

```diff
@@ -198,58 +198,60 @@ class CompanyField:
 MySecondDataClassDataset = Annotated[StructuredDataset, kwtypes(info=InfoField)]
 MyNestedDataClassDataset = Annotated[StructuredDataset, kwtypes(info=kwtypes(contacts=ContactsField))]
 
+image = ImageSpec(packages=["pandas", "tabulate"], registry="ghcr.io/flyteorg")
```

Comment:
Should I add this registry like other example?

Reply:
yes